plugins/inputs/win_wmi/win_wmi.go

```diff
@@ -0,0 +1,268 @@
+//go:build windows
+// +build windows
+
+package win_wmi
+
+import (
+	_ "embed"
+	"fmt"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	ole "github.com/go-ole/go-ole"
+	"github.com/go-ole/go-ole/oleutil"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/filter"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+// Query struct
+type Query struct {
+	Query                string
```

Comment:
Query should not be an exported field as it is used internally.

```suggestion
	query                string
```

Reply:
Done!